<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>538</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>HMI panel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/rs_hmi_logo.png</normaloff>:/rs_hmi_logo.png</iconset>
  </property>
  <widget class="QWidget" name="widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="hmiScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>910</width>
         <height>492</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionOpenCommSettings"/>
    <addaction name="separator"/>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menudebug">
    <property name="title">
     <string>debug</string>
    </property>
    <addaction name="actionTest"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
   <addaction name="menudebug"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenCommSettings">
   <property name="text">
    <string>Связь с контроллером</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionTestMenu">
   <property name="text">
    <string>тест меню</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="text">
    <string>Развернуть на весь экран</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
